src/uct/ib/ud/base/ud_iface.c

```diff
@@ -511,6 +550,9 @@ ucs_config_field_t uct_ud_iface_config_table[] = {
     {"SLOW_TIMER_BACKOFF", "2.0", "Timeout multiplier for resending trigger",
      ucs_offsetof(uct_ud_iface_config_t, slow_timer_backoff),
                   UCS_CONFIG_TYPE_DOUBLE},
+    {"ETH_GID_FILTER_ENABLE", "y",
+     "Enable packets filtering by destination GIDs on an Ethernet network",
```

Comment:
Enable checking destination GID for incoming packets of Ethernet networks. Mismatched packets are silently dropped.

Reply:
Btw,shall we have counter for such drop eventsfor future perf work?

Reply:
Will add in the next PR (for this to be merged asap). Opened issue for that #1855